### ISSUE TITLE
Angular [conroller] causes console error 

### ISSUE THREAD

**Issue by StasSpawn** [nolimits4web/swiper#6529]
Demo with this console error 
https://stackblitz.com/edit/swiper-angular-controller?file=src%2Fapp%2Fapp.component.ts

![2023-03-28_191303](https://user-images.githubusercontent.com/56681850/228306348-4b6ec397-d12d-452b-88bb-4456da0c238e.png)

I have 2 sliders, they controll each other. 
There are a lot of swiper sliders on pages. But this error present only on pages with [conroller] sliders.

Also screenshot of code.
![2023-03-28_192916](https://user-images.githubusercontent.com/56681850/228307045-94c81863-985f-4c0c-b339-75e01e6919f6.png)

Important, that I was testing - create copy of simble slider, gave them [conroller] for each other and the same error appear.

**Comment by StasSpawn** [comment]
Here you can see demo with this console error 
https://stackblitz.com/edit/swiper-angular-controller?file=src%2Fapp%2Fapp.component.ts

**Comment by bitrxjs** [comment]
@StasSpawn 
`https://stackblitz.com/edit/swiper-angular-controller-vevmx8`
does this solve ur issue

**Comment by StasSpawn** [comment]
@bitrxjs 
Unfortunately, no. 
![2023-04-06_205231](https://user-images.githubusercontent.com/56681850/230491528-0ab96de9-7757-4081-9a79-66b04bc7d65d.png)

I've paste this code, add req imports, but error still present.

**Comment by bitrxjs** [comment]
@StasSpawn  
stackblitz link works fine. 
please share template code also

**Comment by StasSpawn** [comment]
@bitrxjs
Hello. I see, that in your example sliders are not cotrolled by each other. They switching separately from each other.
https://stackblitz.com/edit/swiper-angular-controller-vevmx8

**Comment by StasSpawn** [comment]
@bitrxjs
Also, error really dissapeared, when I add corrent code to my site. BUT, both sliders has to switch each other. With your code only one slider switch another.

**Comment by bitrxjs** [comment]
@StasSpawn 
try this
`https://stackblitz.com/edit/swiper-angular-controller-vevmx8`

**Comment by bitrxjs** [comment]
@StasSpawn 
is this what u want
`https://stackblitz.com/edit/swiper-angular-controller-qqjhtv`

**Comment by StasSpawn** [comment]
@bitrxjs Yes, this work poerfectly. 
So, is this some bug of slider library, or it's how it suppose to work?

**Comment by bitrxjs** [comment]
@StasSpawn 
No bug
The angular wrapper have some issue related to updating controller
So i just used the swiperRef instance and called methods on it directly bypassing the angular wrapper.

**Comment by bitrxjs** [comment]
@nolimits4web pls close this one

**Comment by StasSpawn** [comment]
@bitrxjs 
Hello. This fix is working, but I haму a third slider in this scheme. Is ther a possibility to add 3 slider and make them all synchronized? 

I've tried this code, but it couses error: **Uncaught RangeError: Maximum call stack size exceeded**

```
@ViewChild('firstSlider') firstSlider?: SwiperComponent;
  @ViewChild('secondSlider') secondSlider?: SwiperComponent;
  @ViewChild('thirdSlider') thirdSlider?: SwiperComponent;

  ngAfterViewInit() {
    if (this.secondSlider?.swiperRef && this.firstSlider && this.thirdSlider) {
      this.firstSlider.swiperRef.controller.control = [this.secondSlider.swiperRef, this.thirdSlider.swiperRef];
      this.firstSlider.swiperRef.update();

      this.secondSlider.swiperRef.controller.control = [this.firstSlider.swiperRef, this.thirdSlider.swiperRef];
      this.secondSlider.swiperRef.update();

      this.thirdSlider.swiperRef.controller.control = [this.firstSlider.swiperRef, this.secondSlider.swiperRef];
      this.thirdSlider.swiperRef.update();
      ////// END Fix for slider with controller
    }
  }
```

**Comment by StasSpawn** [comment]
@bitrxjs Hello. There another bug. I'm not sure is it related to that feature or not. 

With loop we must use loopedSlides. But if some data.length there it causes error. 

![2023-07-26_190631](https://github.com/nolimits4web/swiper/assets/56681850/f37f31be-0a7d-4240-a089-c86ec3a6484b)
![2023-07-26_191110](https://github.com/nolimits4web/swiper/assets/56681850/19f2a256-7606-4d49-9960-c6979b2c0b4d)


This condition isn't helps.
![2023-07-26_192025](https://github.com/nolimits4web/swiper/assets/56681850/9c081027-b38b-4b4f-be2e-8e45cb0c6496)

If I add *ngIf="slidesData" to parent html block - control just not working. 

So, basically I need fix error and keep slider control.

**Comment by StasSpawn** [comment]
@bitrxjs still need help with this :(
